fontforge/woff.c

```diff
@@ -508,9 +540,9 @@ return( ret );
     if ( sf->woffMetadata!= NULL ) {
 	int uncomplen = strlen(sf->woffMetadata);
 	uLongf complen = 2*uncomplen;
-	char *temp=malloc(complen+1);
+	unsigned char *temp=malloc(complen+1);
 	newoffset = ftell(woff);
-	compress(temp,&complen,sf->woffMetadata,uncomplen);
+	compress(temp,&complen,(unsigned char *)sf->woffMetadata,uncomplen);
 	fwrite(temp,1,complen,woff);
```

Comment:
We might get a warning here about temp not being a plain `char *`, but we're still much better off than before.